README.md

```diff
@@ -47,17 +48,28 @@ import '@warp-ds/elements/components/toast';
 For information on the components available in the package and how to use them,
 see the [Warp Design System documentation](https://warp-ds.github.io/tech-docs/).
 
-
 ## Releases
 
 This project is continuously published to [NPM](https://www.npmjs.com/package/@warp-ds/elements) and [Eik](https://assets.finn.no/pkg/@warp-ds/elements) using a `next` tag (e.g. `1.1.0-next.1`).
 Anyone needing to use the latest changes of this package can point to the `next` version while waiting for the stable release.
 
-
 ## Changelog
 
 Detailed changes for each release can be found in the [CHANGELOG](CHANGELOG.md) file.
 
+## How to test component-classes locally
+
+It is possible to test changes that have been made to component-classes from the [@warp-ds/css](https://github.com/warp-ds/css) package but that has not yet been published to Eik. You simply need to link the `@warp-ds/css` package in this repository and add `@unocss-placeholder` inside the static styles of the component that you want to test:
+
+```js
+static styles = [
+    css`
+      @unocss-placeholder
+      `,
+  ]
+```
+
+Once you have tested it locally, make sure to remove the `@unocss-placeholder` from the static styles before pushing any changes to the repository.
```

Comment:
Sounds great! Would it make sense to suggest to run `pnpm dev` before the `Once you have tested...`? Something like:
```
Run `pnpm dev` to see the linked styles in action.
```

Reply:
Good point! I’ll add that :)